.github/workflows/release_build.yml

```diff
@@ -0,0 +1,44 @@
+name: Release build
```

Comment:
Maybe the name "Package and documentation" would describe this flow better. 

(could be also Package**s** and documentation)

Reply:
Renamed to "Build release packages and documentation".